samples/Build2019Demo.Unity/ProjectSettings/ProjectVersion.txt

```diff
@@ -0,0 +1 @@
+m_EditorVersion: 2018.3.11f1
```

Comment:
>2018.3.11f1 [](start = 17, length = 11)

I think the other project is on 2018.3.14f1 currently, might be nice to keep these in sync